source/_integrations/thethingsnetwork.markdown

```diff
@@ -16,6 +16,7 @@ ha_integration_type: integration
 
 <div class='note warning'>
   This integration only supports TTNv2 that has been definitively switched off in 2022 (deprecated).
+  For TTNv3 use the MQTT-Integration.
```

Comment:
Should this link to `/integrations/mqtt`?

Reply:
Yes it can be linked.